OOVPA.h

```diff
@@ -98,10 +104,10 @@ typedef struct _LOVP
 #define XREF_ENTRY( Offset, XRef)    \
     { Offset, XRef }
 
-// UNUSED Macro for storing a normal (non-XRef) {Offset, Value}-Pair
+// Macro for storing a normal (non-XRef) {Offset, Value}-Pair
 // Offsets can go up to 16 bits, values are always one byte (8 bits)
 #define OV_ENTRY(Offset, Value)    \
```

Comment:
#62 suggests to use `OV_ENTRIES`, but also `OV_ENTRY`. I'd recommend to only use one name, which can handle 1-N arguments.

```
MATCH(0x0000, 0x01)
MATCH(0x0000, 0x01, 0x02)
```

should work; so there's no benefit doing (it will only be confusing and make writing additional parsers harder):

```
MATCH1(0x0000, 0x01)
MATCHn(0x0000, 0x01, 0x02)
```